framework/src/st_things/things_stack/framework/things_data_manager.h

```diff
@@ -129,6 +129,19 @@ bool dm_is_es_complete(void);
 bool dm_get_easy_setup_use_artik_crt(void);
 char *dm_get_mnid(void);
 
+/** This API will return the access token of the cloud
+ * 
+ * First it will get the cloud credentials from provisioning.dat file and
+ * then copies the value of access token in cloud_access_token and will return cloud_access_token.
```

Comment:
Refactor comments like below: 
1. No need to state the obvious : This API / This function  / This method.
2. No need to reveal platform specific / internal implementation details like file name.


Consider wording like below 

---------------
Returns access token received from cloud.

Internally loads the cloud information stored in persistent storage,
retrieves access token from it and returns the copy of the same. 

*Note: Memory allocated for return parameter should be released by the caller.
-----------------